package_linter.py

```diff
@@ -395,7 +395,7 @@ def check_helper_consistency(path, script_name, script_path):
     check if ynh_install_app_dependencies is present in install/upgrade/restore
     so dependencies are up to date after restoration or upgrade
     """
-    script = open(script_path).read()
+    script = read_file(script_name)
```

Comment:
Comments are already removed here https://github.com/YunoHost/package_linter/blob/master/package_linter.py#L88-L91
We shall remove it as this method is more elegant.

Reply:
Uuuuh, I don't get it - yes I use `read_file` to remove comments and stuff in order to remove comments, as suggested by zamentur ? o.O